src/screens/FeedScreen/RenderList.js

```diff
@@ -47,6 +48,9 @@ const RenderListFeed = (props) => {
     source = SOURCE_FEED_TAB,
     hideThreeDot = true,
     showAnonymousOption = false,
+    onDeletePost,
+    isShowDelete,
+    isSelf,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props `onDeletePost`, `isShowDelete`, and `isSelf` are added. These seem to be used for handling post deletion and checking if the current user is the author of the post. If these props are properly passed and used, then these changes are good.
```diff
+     onDeletePost,
+     isShowDelete,
+     isSelf,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->